specification/authorization/resource-manager/Microsoft.Authorization/preview/2018-01-01-preview/authorization-ClassicAdminCalls.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "AuthorizationManagementClient",
-    "version": "2015-07-01",
```

Comment:
>"2015-07-01", [](start = 15, length = 13)

classicadmin API is owned by ARM team,hence we didnt modify the api version here and modified only for the API calls which we own